tests/test_huggingface_flash.py

```diff
@@ -122,6 +164,7 @@ def test_flash2(model_name: str, use_flash_attention_2: bool):
             'config_overrides': {
                 'num_hidden_layers': 2,
                 'intermediate_size': 64,
+                'use_cache': False,
```

Comment:
cache needs to be false when using patch https://github.com/mosaicml/llm-foundry/blob/main/llmfoundry/models/layers/llama_attention_monkeypatch.py#L83

Reply:
discussed offline, this should be removed